components/raftstore/src/store/worker/region.rs

```diff
@@ -277,16 +281,23 @@ where
         last_applied_state: RaftApplyState,
         kv_snap: EK::Snapshot,
         notifier: SyncSender<RaftSnapshot>,
+        for_balance: bool,
     ) {
         SNAP_COUNTER.generate.all.inc();
         let start = tikv_util::time::Instant::now();
+        let _ = WithIOType::new(if for_balance {
```

Comment:
I made a similar mistake recently. `_` will actually be dropped immediately. Use `_io_type_guard` instead.

Reply:
TIL